hera_cal/lstbin_simple.py

```diff
@@ -0,0 +1,1740 @@
+"""
+An attempt at a simpler LST binner that makes more assumptions but runs faster.
+
+In particular, we assume that all baselines with a particular observation have the same 
+time and frequency arrays, and use this to vectorize many of the calculations.
```

Comment:
Yes, this will break with BDA. I think with BDA we should be easily able to put in another loop over BDA groups and maintain the overall vectorization here. I'll make an issue.